packit_service/utils.py

```diff
@@ -253,3 +258,34 @@ def pr_labels_match_configuration(
         not configured_labels_absent
         or all(label not in pr_labels for label in configured_labels_absent)
     )
+
+
+def download_file(url: str, path: Path):
```

Comment:
We have pretty much the same code in [packit](https://github.com/packit/packit/blob/2e75e6ff4c0cadb55da1c8daf9315e4b0a69e4a8/packit/base_git.py#L566-L583), shouldn't we unify it?

Reply:
Perhaps we could also use some library (pycurl?) to make downloading more robust. Anyway, I think this can be a follow-up issue.

Reply:
I will have a thought about it. Using a library sounds like a good idea :+1:

Reply:
added a TODO note to the code for now